examples/blog_example/move/Move.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "RoochDemo"
```

Comment:
Named `RoochDemo`  to `RoochExampleDemo` ?

Reply:
or RoochBlogDemo

Reply:
Already modified.